src/Microsoft.Health.Fhir.Core/Features/KnownQueryParameterNames.cs

```diff
@@ -55,5 +55,7 @@ public static class KnownQueryParameterNames
         public const string Id = "_id";
 
         public const string Type = "_type";
+
+        public const string BirthDate = "birthdate";
```

Comment:
What is special about birth date?

Reply:
@brendankowitz how would I accept any supported sort parameter?

Reply:
"Supported" here means that we know that the search parameter can have at most one value, because the current implementation does not use the smallest value as the value to sort by for a resource.
We can (1) use a hard-coded list _for now_, (2) generate the set of supported search parameters by analyzing the FHIRPath expressions against the structure definitions, (3) support all search parameters, regardless of multiplicity.
I am okay with (1) for now, but I would not put the list in this file, since it is a SQL-only feature.